### PR TITLE
Clean duplicate code in redis

### PR DESCRIPTION
**Pull request by wbc6080**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

When adding device data to the redis database of mapper-framework, it will be divided into two situations: whether the order set exists or not. But in fact, the code is exactly the same in both cases, so the duplicate code is cleaned up in this PR

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: staging/src/github.com/kubeedge/mapper-framework/_template/mapper/data/dbmethod/redis/client.go [hunk 1]
```diff
@@ -69,33 +69,16 @@ func (d *DataBaseConfig) AddData(data *common.DataModel) error {
 	// The key to construct the ordered set, here DeviceName is used as the key
 	klog.V(1).Infof("deviceName:%s", data.DeviceName)
 	// Check if the current ordered set exists
-	exists, err := RedisCli.Exists(ctx, data.DeviceName).Result()
+	deviceData := "TimeStamp: " + strconv.FormatInt(data.TimeStamp, 10) + " PropertyName: " + data.PropertyName + " data: " + data.Value
+	// Add data to ordered set. If the ordered set does not exist, it will be created.
+	_, err = RedisCli.ZAdd(ctx, data.DeviceName, &redis.Z{
```
Comment (WillardHu) [inline staging/src/github.com/kubeedge/mapper-framework/_template/mapper/data/dbmethod/redis/client.go:74]: Does the key need a namespace? Do we need to add another recognizable prefix?

Comment (wbc6080) [reply staging/src/github.com/kubeedge/mapper-framework/_template/mapper/data/dbmethod/redis/client.go:74]: Yes, the prefix here needs to be replaced by deviceID. I will submit another PR to modify the influx、redis、mysql 、tdengine uniformly.

**Review by Shelley-BaoYue — COMMENTED** [review]
/approve

**Pull request merged**